src/kafka-connect/create-connector/create-connector.controller.js

```diff
@@ -31,118 +31,171 @@ angularAPP.controller('CreateConnectorCtrl', function ($scope, $rootScope, $http
   $scope.isDisabledTab = function(index) { return (index == $scope.selectedTabIndex) ? 'false' : 'true'; }
 
 
+  $scope.aceLoaded = function (_editor) {
+    console.log("Ace for create-connector loaded");
+    $scope.editor = _editor;
+    $scope.editor.$blockScrolling = Infinity;
+    $scope.acePropertyFileSession = _editor.getSession();
+    var lines = 15;//$scope.connectorDetails.connectorDetailsInString.split("\n").length;
+    _editor.setOptions({
+      minLines: lines,
+      maxLines: lines,
+      highlightActiveLine: false
+    });
+  };
+
 
   //If user changes config from the editor
   $scope.$watch('formValuesPerSection', function() {
  if ($scope.formValuesPerSection) {
-      console.log('test', $scope.formValuesPerSection)
+      //console.log('test', $scope.formValuesPerSection)
       $scope.formValuesPerSection = $scope.formValuesPerSection.replace("\r", "");
-      console.log('test', $scope.formValuesPerSection)
+      //console.log('test', $scope.formValuesPerSection)
       var flatValuesArray = $scope.formValuesPerSection.split("\n");
-      $scope.curlCommand = NewConnectorFactory.getCurlCommand(flatValuesArray);
+      validateConnectorFn();
   }
   });
 
 
-  //TODO save & Validate = duplicate code
-
-    $scope.validateConnector = function (object) {
-
-            flatValuesArray = $scope.formValuesPerSection.split('\n');
+    function validateConnectorFn () {
```

Comment:
this function becomes huge? maybe split it in smaller functions so it's at least readable?